content/en/continuous_integration/troubleshooting.md

```diff
@@ -171,9 +171,9 @@ If you have admin access, you can update it from the [Repository Settings Page][
 2. Make sure the Datadog Agent host is properly configured and is reachable by the Datadog Plugin. You can test connectivity by clicking on the **Check connectivity with the Datadog Agent** button on the Jenkins plugin configuration UI.
 3. Check for any errors in the Jenkins logs. You can enable debug-level logs for the Datadog plugin by [creating a `logging.properties` file][1] and adding the line: `org.datadog.level = ALL`.
 
-### Pipeline not found
+### Pipeline execution view not found
 
-A "Pipeline not found" message is shown when you click on incomplete data coming from an in-progress pipeline. Data is received progressively for stages, jobs, or custom commands. Wait until the pipeline has finished and try again.
+If you click on a pipeline execution with running status, a "Pipeline execution was running more than 3 days" message is shown. Data is received progressively for stages, jobs, or custom commands. Wait until the pipeline has finished and try again.
```

Comment:
I don't understand this sentence. If it's not found, is it because it has been running for more than 3 days?

Reply:
Actually, this should not change.

Reply:
Actually, this must not change as this can still happen for other CI Providers. I've changed again to the initial paragraph, with a link to the `Supported Features`.